src/SetupApi.Desktop/SetupApi.Helpers.cs

```diff
@@ -79,20 +79,20 @@ public static partial class SetupApi
         }
 
         public static string SetupDiGetDeviceInterfaceDetail(
-            SafeDeviceInfoSetHandle lpDeviceInfoSet,
-            DeviceInterfaceData oInterfaceData,
-            IntPtr lpDeviceInfoData)
+            SafeDeviceInfoSetHandle deviceInfoSet,
+            DeviceInterfaceData interfaceData,
+            DeviceInfoData deviceInfoData)
         {
             var requiredSize = new NullableUInt32();
 
             // First call to get the size to allocate
             SetupDiGetDeviceInterfaceDetail(
-                lpDeviceInfoSet,
-                ref oInterfaceData,
+                deviceInfoSet,
+                ref interfaceData,
                 IntPtr.Zero,
                 0,
                 requiredSize,
-                null);
+                deviceInfoData);
```

Comment:
I see this argument was missing only from the first call (the "how big should the buffer be?" call). So perhaps that's by design?